server/src/migrations/20220310091901_integrationsRework.ts

```diff
@@ -0,0 +1,66 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('integration', (table) => {
+    table
+      .integer('tokenId')
+      .references('id')
+      .inTable('integration')
+      .onDelete('SET NULL')
+      .index();
+
+    table.string('trelloTableId').nullable();
+  });
+
+  await knex.schema.createTable('trelloColumnMappings', (table) => {
+    table.increments('id').primary();
+    table
+      .integer('integrationId')
+      .references('id')
+      .inTable('integration')
+      .onDelete('CASCADE')
+      .index();
+
+    table.string('fromColumn', 75).notNullable();
+    table.enu('toStatus', ['NOT_STARTED', 'IN_PROGRESS', 'COMPLETED'], {
```

Comment:
I wonder if it would be better after all to keep this just an integer on the db, since that is how we have stored all the enums so far.

It would also simplify adding new variants (as long as the old ones are not changed), without migrations.